src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
After looking deeper to the code I have to "ammend" my previous thought :)

The functions that interact with MongoDB (i.e. the ones that are going to use connectionOperations and safeBson functions) should remain here. But there is logic currently in this module that has nothing to do with DB interaction (e.g. the function to create the refresh thread).

I don't suggest to move anything in this PR but probably mark with a FIXME the candidate functions to be moved is a good idea.